dashboard/app/controllers/registrations_controller.rb

```diff
@@ -143,9 +145,11 @@ def parent_email_params
       require(:user).
       tap do |user|
         user[:parent_email_preference_email] = user[:parent_email]
-        user[:parent_email_preference_opt_in_required] = '1'
-        user[:parent_email_preference_request_ip] = request.ip
-        user[:parent_email_preference_source] = EmailPreference::PARENT_EMAIL_CHANGE
+        user[:parent_email_preference_opt_in_required] = '0'
```

Comment:
Have you considered doing this on the UI side? If Yes/No is clicked, then set the hidden value to '1'?

Reply:
I could just do that here. I might rename this variable if we go that route -- I don't think "opt_in_required" makes sense anymore. Maybe `parent_email_preference_opt_in_update_required` or something